praca/praca.tex

```diff
@@ -134,7 +134,7 @@ \section{Zajętość MOP-ów i dostępne usługi}
 	\item Jaka będzie zajętość miejsc parkingowych za~godzinę?
 \end{enumerate}
 
-Odpowiedzią na te pytania będzie \textbf{Mopsik} (MOP -- System Informowania Kierowców) -- aplikacja mobilna i strona internetowa.
+Odpowiedzią na te pytania będzie \textbf{Mopsik} (MOP -- System Informowania Kierowców) -- \hyperref[mopsik_mobile]{aplikacja mobilna} i \hyperref[mopsik_web]{strona internetowa}. Programy te pobierają dane z API wystawianego przez \hyperref[mopsik_api]{Mopsik--Serwer}.
```

Comment:
Nie powinniśmy już pisać "jest" zamiast "będzie"? Ale to można poprawić do czwartku.